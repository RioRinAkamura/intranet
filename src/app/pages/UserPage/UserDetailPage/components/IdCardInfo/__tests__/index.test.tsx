import * as React from 'react';
import { render } from '@testing-library/react';

import { IdCardInfo } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<IdCardInfo  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<IdCardInfo />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
