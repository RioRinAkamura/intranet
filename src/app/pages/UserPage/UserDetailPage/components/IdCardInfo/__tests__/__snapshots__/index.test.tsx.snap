// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IdCardInfo  /> should match snapshot 1`] = `
.c0 {
  color: #1890ff;
  margin-bottom: 24px;
}

<div
  class="c0"
>
  <b>
     
    UserDetailPage.modal.form.profile.idCard.title
    Id Card Information
  </b>
</div>
`;
